hooks/useCollectionTwapBidChange/useCollectionTwapBidChange.ts

```diff
@@ -0,0 +1,66 @@
+import { useConfig } from 'hooks/useConfig';
+import { useOracleInfo } from 'hooks/useOracleInfo';
+import { OraclePriceType } from 'lib/oracle/reservoir';
+import { percentChange } from 'lib/tokenPerformance';
+import { useMemo, useRef } from 'react';
+import {
+  LatestTwabForCollectionBeforeTimeQuery,
+  LatestTwabForCollectionBeforeTimeDocument,
+} from 'types/generated/graphql/twabs';
+import { useQuery } from 'urql';
+
+export function useCollectionTwapBidChange(collection: string) {
+  const { twabsApi } = useConfig();
+  const now = useRef(new Date());
```

Comment:
had to use refs or else `useQuery` would infinite loop and always compute new dates

Reply:
https://github.com/trojanowski/react-apollo-hooks/issues/198

Reply:
I think you might be able to encapsulate this and `twentyFourHoursAgo` into a single useMemo but not critical